foundations/javascript_basics/object_basics.md

```diff
@@ -99,7 +107,9 @@ Consider the array from our previous example, `[1, 2, 3, 4, 5]`.
 If we wanted to remove all even numbers from this array, we could use `.filter()` like this:
 
 ```javascript
-arr.filter(isOdd); // Outputs [1, 3, 5];
+const filteredArr = arr.filter(isOdd);
+console.log(filteredArr) // Outputs [1, 3, 5];
+console.log(arr) // Outputs [1, 2, 3, 4, 5], original array is not affected
```

Comment:
```suggestion
const oddNums = arr.filter((num)) => num % 2 !== 0);
console.log(oddNums) // Outputs [1, 3, 5];
console.log(arr) // Outputs [1, 2, 3, 4, 5], original array is not affected
```
Since `isOdd` hasn't been defined, we could follow the point in the `map` example of defining an inline callback.
The variable names can then be changed for clarity.